backend/src/src-cvesearch/src/test/java/org/eclipse/sw360/cvesearch/datasource/matcher/ModifiedLevenshteinDistanceTest.java

```diff
@@ -12,6 +12,8 @@
 import org.junit.Test;
 
 import static org.eclipse.sw360.cvesearch.datasource.matcher.ModifiedLevenshteinDistance.levenshteinMatch;
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.*;
 
 public class ModifiedLevenshteinDistanceTest {
```

Comment:
I see that the behaviour of `levenshteinMatch` has changed so that the needle must match the haystack in at least one character, or else `Integer.MAX_VALUE` is returned and also the characters' case is ignored now, but neither of these changes is reflected in javadoc comments or in the tests. This Levenshtein distance trickery is not the most self-explanatory code in the project. I wish you'd update the documentation on what exactly is modified from the standard Levenshtein distance.